packages/unity-interface/BrowserInterface.ts

```diff
@@ -686,6 +697,57 @@ export class BrowserInterface {
     }
   }
 
+  public async CreateChannel(createChannelPayload: CreateChannelPayload) {
```

Comment:
No need for async when using .catch
```suggestion
  public CreateChannel(createChannelPayload: CreateChannelPayload) {
```